### PR TITLE
newdocs 1.2.29

### PR DESCRIPTION
**Pull request by taildigital**
Adding new docs ver 1.2.29

### CODE REVIEW

**Review by dsaitotail — APPROVED** [review]
ok to go

**Comment by dsaitotail** [comment]
ok

**Review by dsaitotail — CHANGES_REQUESTED** [review]
req changes

**Review by dsaitotail — APPROVED** [review]

**Comment by dsaitotail** [comment]
approved

**Review by dsaitotail — APPROVED** [review]
aproved this

File: README.md [hunk 1]
```diff
@@ -910,6 +910,148 @@ public class MainActivity extends AppCompatActivity implements ActivityCompat.On
 
 </pre>
 
+## Events
+
+### Save Events 
+
+Events are key / value data that can be saved when an in-app action occurs. ex: User clicked to buy a product, User cancel a purchase, etc.
```
Comment (dsaitotail) [inline README.md:917]: rv

**Review by dsaitotail — APPROVED** [review]

**Comment by taildigital** [comment]
not ok

**Pull request closed without merging**